### PR TITLE
Add linters html css

### PR DESCRIPTION
**Pull request by sireclifford**
The following linters were added to correct the codes;

- Lighthouse:-An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more.
- Webhint:-A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
- Stylelint:-A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.

**Tested and working**

Screenshot 2022-05-24 at 9 34 58 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
-# Hello-world
\ No newline at end of file
+# Hello World
```
Comment (Ismailco) [inline README.md:1]: - [ ] This Pull Request should have a descriptive title telling you what changes are inside, you can add a description to this PR with a summary describing introduced changes in more detail

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @sireclifford,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding more info to your README file so it would be descriptive and more professional 
- You could consider adding the ```.DS_Store``` file to the ```.gitignore``` file

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sireclifford) [reply README.md:1]: @Ismailco thank you for your review. I do not understand how to add description to the pull request. Can you direct me on how to do that please? Thank you

Comment (Ismailco) [reply README.md:1]: @sireclifford, you can click on the ```...``` at the top right of the first comment in this PR and choose ```edit``` 
|||
|--|--|
|Screenshot_20220524_220401|Screenshot_20220524_175736|

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi @sireclifford,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**